ow_plexil/src/plans/FaultHandlingPattern5.plp

```diff
@@ -0,0 +1,104 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Pattern summary: Execute a sequence of actions. If a fault is injected
+// during a related action, the action will stop and end in the ABORTED state. 
+// As long as the action is stopped, a backup action will repeat. If/when the 
```

Comment:
Ah yes, that is a little unclear. Thanks for pointing it out! In this case, I am considering panning to be a "backup" to the arm actions unstow or stow in Mission. So if Unstow or Stow are interrupted, panning should happen until Unstow or Stow can be restarted, i.e. when there are no longer any active arm faults. So, in more general terms, perhaps it would make more sense to say "As long as the action is stopped, another unrelated action will occur continuously" or something along those lines?

Reply:
Oh I see now! Your presentation today helped clear this up too. So it sounds like backup actions are necessarily different from the primary action, and are just meant to give the lander something to do while the fault is resolved? Thanks for explaining!

Reply:
You got it!